packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -95,6 +96,36 @@ const start = async (zcf, privateArgs) => {
   /** @type {ZCFMint} */
   const runMint = await zcf.registerFeeMint('RUN', feeMintAccess);
 
+  const { brand: runBrand } = await E(runMint).getIssuerRecord();
+  const { publication: metricsPublication, subscription: metricsSubscription } =
+    makeSubscriptionKit();
+
+  // shortfall in Vaults due to liquidations less than debt. This value can be
+  // reduced by various actions which burn IST.
```

Comment:
until we rename, might be more confusing to refer to it in two ways
```suggestion
  // reduced by various actions which burn RUN.
```

Reply:
done